test/NuGet.Core.Tests/NuGet.Common.Test/LocalizedTestCollection.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Globalization;
+using Xunit;
+
+namespace NuGet.Common.Test
+{
+    [CollectionDefinition(TestName, DisableParallelization = true)]
+    public class LocalizedTestCollection
+    {
+        public const string TestName = "LocalizedTests";
+
+        public static CultureInfo DefaultCurrentUICulture;
+
+        public static CultureInfo DefaultCurrentCulture;
```

Comment:
I think these two properties can also be `private` members.

Reply:
Fixed.